extract-initramfs-from-vmlinuz.sh

```diff
@@ -71,34 +71,33 @@ trap "rm -rf ${tmp}" EXIT
 tmp_dec="${tmp}/decompress"
 mkdir "${tmp_dec}"
 fr_prefix="${tmp}/first-round"
-try_unzstd_decompress "${image}" "${tmp_dec}" "${fr_prefix}"
 
-shopt -s failglob
+perform_round() {
+    local image="${1}"
+    local tmp_dec="${2}"
+    local round_prefix="${3}"
+    try_unzstd_decompress "${image}" "${tmp_dec}" "${round_prefix}"
+    for rnd in "${round_prefix}"*; do
+        if [[ $(file --brief "${rnd}") =~ 'cpio archive' ]]; then
+            mkdir -p "${out}/rootfs-${rootfs_idx}"
+            while cpio --quiet --extract --make-directories --directory="${out}/rootfs-${rootfs_idx}" --nonmatching 'dev/*'; do
+                rootfs_idx=$(( rootfs_idx + 1 ))
+                mkdir -p "${out}/rootfs-${rootfs_idx}"
+            done <${rnd}
+            rmdir "${out}/rootfs-${rootfs_idx}"
+        fi
+    done
+}
 
+shopt -s nullglob
 rootfs_idx=0
```

Comment:
I'd like to have two changes - `rootfs_idx` kinda became a global variable that gets changed in "random" places (inside the `perform_round` function), so could you:
- move this line before the `perform_round` function definition
- make it upper case

And the second change - could you update the comment at the top of the file to say something like "This will create one or more out-dir/rootfs-N directories, and one of them will contain initramfs."

Thanks.

Reply:
Just read my comment again, and realized it's poorly worded in one place. To make it clear - the uppercasing was about making `rootfs_idx` a `ROOTFS_IDX`.

Reply:
got it, pushed a fixed version, take a look